src/context/cart-context.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, createContext, useContext, useReducer } from "react";
+import axios from "axios";
+
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+   //fetching all product data from server
+   useEffect(() => {
+      (async () => {
+         try {
+            const response = await axios.get("/api/products");
+            dispatch({
+               type: "UPDATE_DATA",
+               payload: response.data.products,
+            });
+         } catch (error) {
+            console.log(error);
```

Comment:
fix: remove console.log

Reply:
Will remove it, thanks for pointing it out